lisa/sut_orchestrator/libvirt/qemu_platform.py

```diff
@@ -39,6 +40,20 @@ def _create_node_os_disk(
         self, environment: Environment, log: Logger, node: Node
     ) -> None:
         node_context = get_node_context(node)
+        if environment.runbook.nodes_requirement:
+            node_req: List[NodeSpace] = environment.runbook.nodes_requirement
+            for _, node_space in enumerate(node_req):
+                node_runbook: BaseLibvirtNodeSchema = node_space.get_extended_runbook(
+                    self.node_runbook_type(),
+                    type_name=type(self).type_name(),
+                )
+                if node_runbook.disk_img_resize_gib:
+                    qemu_tool = self.host_node.tools[QemuImg]
+                    src_file = node_context.os_disk_base_file_path
```

Comment:
Also, as @cwize1 mentioned earlier, let's not resize the `os_disk_base_file_path`. 
diffs do not support the resize, so we will need to copy the `os_disk_base_file_path` to `os_disk_file_path` (just how we do in ch platform) and resize.
@anirudhrb  any comments on this ?

Reply:
resize is on os_disk_file_path.

Reply:
Resize worked on diff image as well. I am resolving this comment